clusterctl/README.md

```diff
@@ -22,13 +22,10 @@ $ go build
 
```

Comment:
Can you add a _TBD_ here so that it's clear that the limitations section isn't filled out? Right now it is easy to think that Creating a cluster falls in the limitations section.

Reply:
Done